packages/alfa-dom/src/h.ts

```diff
@@ -104,6 +120,34 @@ export namespace h {
     );
   }
 
+  export function shadow(children: Array<Node | string>): Shadow;
+  export function shadow(
+    mode: Shadow.Mode,
+    children: Array<Node | string>,
+    style: Array<Sheet>
+  ): Shadow;
+  export function shadow(
+    childrenOrMode: Array<Node | string> | Shadow.Mode,
+    children?: Array<Node | string>,
+    style: Array<Sheet> = []
+  ): Shadow {
+    let mode: Shadow.Mode;
+    if (Array.isArray(childrenOrMode)) {
+      children = childrenOrMode;
+      mode = Shadow.Mode.Open;
+    } else {
+      mode = childrenOrMode;
+    }
+
+    return Shadow.of(
+      mode,
+      children!.map((child) =>
+        typeof child === "string" ? text(child) : child
+      ),
+      style
+    );
+  }
+
```

Comment:
This turned very complex very fast 🙈 I do see your point in the mode being less important than the children, and possibly the style, so let's take the chance to make it the last parameters in both `Shadow.of()` and `h.shadow()`. The coming release will contain breaking changes either way.